cf/i18n/resources/fr_FR.all.json

```diff
@@ -2665,8 +2665,8 @@
       "modified": false
    },
    {
-      "id": "List all apps in the target space",
-      "translation": "Liste de toutes les applications dans l'espace ciblé",
+      "id": "List all apps in space of targeted organization",
+      "translation": "List all apps in space of targeted organization",
```

Comment:
Here is the new translation, please use it: `Liste de toutes les applications dans l'espace de l'organisation ciblé`